netmaster/master/netmaster_test.go

```diff
@@ -688,3 +703,94 @@ func TestGetAllocatedIPs(t *testing.T) {
 		log.Fatalf("got networks '%s' expected '%s'", networks, expectedAllocedIPs)
 	}
 }
+
+func verifyIPAlloc(networkID, expIPs string) {
+	nwCfg := &mastercfg.CfgNetworkState{}
+	nwCfg.StateDriver = fakeDriver
+	if err := nwCfg.Read(networkID); err != nil {
+		log.Fatalf("unable to locate network: %s", networkID)
+	}
+
+	allocIPs := ListAllocatedIPs(nwCfg)
+	if allocIPs != expIPs {
+		log.Fatalf("Got addresses '%s' expected '%s'", allocIPs, expIPs)
+	}
+}
+
+func TestEPCreate(t *testing.T) {
```

Comment:
We should deprecate tests based on old style intent.
It might be better to move this test to objApi_test.go which already has logic to setup resource manager etc.